metaflow/plugins/datatools/s3/s3.py

```diff
@@ -759,7 +759,9 @@ def _info(s3, tmp):
                 "metadata": resp["Metadata"],
                 "size": resp["ContentLength"],
                 "last_modified": get_timestamp(resp["LastModified"]),
-                "encryption": resp["ServerSideEncryption"],
+                "encryption": resp["ServerSideEncryption"]
```

Comment:
can you do `resp.get("ServerSideEncryption")` instead?

Reply:
Can you also add a comment here re: why `ServerSideEncryption` might be missing and under what circumstances?

Reply:
Done.